charts/chirpstack/templates/postgres/configmap.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if not .Values.postgres.existingPsqlUrl -}}
+apiVersion: v1
+kind: ConfigMap
+metadata: {{- include "chirpstack.commonMetadata" (dict "context" . "component" "postgres") | nindent 2 }}
+data:
+  001-init-chirpstack.sh: |
+    #!/bin/bash
+    set -e
+
+    psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" <<-EOSQL
+        create role $CS_POSTGRESQL_USER with login password '$CS_POSTGRESQL_PASSWORD';
+        create database $CS_POSTGRESQL_DB with owner $CS_POSTGRESQL_USER;
+    EOSQL
```

Comment:
Is this an extra user/db? Otherwise, aren't there any flags to auto-create this?

But this is less important than https://github.com/teutonet/teutonet-helm-charts/pull/475#discussion_r1275830328